### PR TITLE
[full-ci][tests-only]Skip scenario related to occ command in ocis

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description

This PR skips tests related to the issue https://github.com/owncloud/ocis/issues/1328 for OCIS as these will be implemented for spaces but not like they are implemented in OC10 now.

## Related Issue

- [Check closed issue No command equivalent to occ still exist in expected-to-failure ](https://github.com/owncloud/ocis/issues/4679)
- [Check closed issue [oCIS] Received share cannot be deleted/unshared if not shared with full permissions still exist in expected-to-failure](https://github.com/owncloud/ocis/issues/4683)

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by amrita-shrestha** [comment]
wait until this PR merged in middleware repo: https://github.com/owncloud/owncloud-test-middleware/pull/124

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:2]:
```suggestion
# this scenario is skipped on ocis because it makes request to shared folder in root (All Files) which in not possible in OCIS
  # but it works for OC10 see issue https://github.com/owncloud/web/issues/6896 for more detail
```
nothing wrong with the comment you've put but rest of the files have this comment that's why

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:33]: Maybe this comment should be above the scenario the below step is for opening the share dialog which technically should be possible in ocis

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:43]: similar here above the scenario and maybe this will be clearer?
```suggestion
#   Configuration for sharing only within the group member is not available in ocis and won't be implemented in the same way as OC10 see issue https://github.com/owncloud/ocis/issues/1328 for more detail
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:58]: same here

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:33]: I'm asking this because when I saw the comment I thought it was for the step below 😕

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:2]: does it matter?

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:33]: I am saying  it doesn't matter because I thought i clearly mentioned **shares only with groups members** which is connected with above step only

Comment (kiranparajuli589) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:33]: The comment in between the scenario is misleading IMHO too.

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (SwikritiT) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:2]: I don't think so I'm just suggesting the changes it's eventually up to you to decide if you want to take it or leave it.

Comment (SwikritiT) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:2]: But I would suggest you at least put the link of the issue or issue tag. It's up to you though

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:43]: I put this comment in the wrong place in this scenario.
About this comment  `#   shares only with groups members is not available now in the oCIS webUI` i was just following the existing pattern `# Reshare permission is not available now in the oCIS webUI`

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:58]: About this comment # shares only with groups members is not available now in the oCIS webUI i was just following the existing pattern # Reshare permission is not available now in the oCIS webUI

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature
Comment (SagarGi) [inline tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature:206]: can you adjust intendation?

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:43]: there is an related issue tag **@issue-ocis-1317** at the top of the feature file so I didn't mention it in the comment.I hope this works

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: I put comment at the top because all scenario was skipped because of the same reason

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: I didn't mention the issue link because at the top **@issue-ocis-1317** tag is present which is related to issue

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:41]: I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:57]: I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature:2]: mentioned comment at the top of the feature  film because this feature file is related to `accept/decline shares coming from internal users` that means somehow this comment will be connected

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature:205]: I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue

File: tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature
Comment (amrita-shrestha) [inline tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature:2]: I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/29048/22/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L33.png
  


</p></details>

File: tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md
Comment (saw-jan) [inline tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md:37]:
```suggestion
-   [webUISharingAcceptShares/acceptShares.feature:33](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L33)
```

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (saw-jan) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: You could add issue tag to https://github.com/owncloud/web/issues/6896

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (saw-jan) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: Can we skip this whole feature file? The features mostly test the admin-sharing-settings (uses `occ`)
```suggestion
```

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature
Comment (saw-jan) [inline tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature:32]: You could add an issue tag (https://github.com/owncloud/web/issues/6896)
```suggestion
  @issue-2512 @issue-4102 @issue-6896 @skipOnOCIS
```

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature
Comment (saw-jan) [inline tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature:205]: wasn't available previously either (AFAIK)
```suggestion
#   exclude-groups-from-sharing configuration is not available in the oCIS
```

File: tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature
Comment (saw-jan) [inline tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature:2]: wasn't available previously either (AFAIK)
```suggestion
# exclude-groups-from-sharing configuration is not available in the oCIS
```

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: @saw-jan  (https://github.com/owncloud/web/issues/6896) and  (https://github.com/owncloud/web/issues/1317) sre inter-related so its better to put only one issue tag

File: tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature:2]: @saw-jan I think `now` is two edge sword. This configuration will be added later on ocis but for now it is not available

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: I also thought the same while modifying this file. But this scenario passes because by default  `shareapi_only_share_with_membership_groups` is off currently in ocis. From my side solution is either we skip the whole scenario doesn't matter if it passes or fails. Another solution, let the passing scenario run. 
```feature

  Scenario: Do not restrict users to only share with groups they are member of
    Given the setting "shareapi_only_share_with_membership_groups" of app "core" has been set to "no" in the server
    When the user shares folder "simple-folder" with group "grp2" as "Viewer" using the webUI
    And user "Carol" accepts the share "Shares/simple-folder" offered by user "Brian" using the sharing API in the server
    Then as "Carol" folder "/Shares/simple-folder" should exist in the server
```

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (saw-jan) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: I don't think they are interrelated. https://github.com/owncloud/web/issues/6896 is about share-jail and https://github.com/owncloud/ocis/issues/1317 is about ability to set various admin settings (like `occ` in oC10)

File: tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature
Comment (saw-jan) [reply tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature:2]: Oh, I read it differently. But I think suggestions are good though :smile:

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: sorry I mean adding both issue links and tags overloads the feature with information.

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (saw-jan) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: This scenario should be covered somewhere for oCIS too. Looks like it is just to share and accept the share (so there might be whole lot of tests that covers it).
As I mention, the main purpose here is to test admin setting. So, maybe we can skip this also.

CC @phil-davis

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (saw-jan) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: Yeah, indeed we should not cluster scenario with tags but since both issues are different and both related to the scenario, it would be good to add them both, specially #6896

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]:
```suggestion
#   shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:41]:
```suggestion
#   shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:57]:
```suggestion
#   allow-shares-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature
Comment (SwikritiT) [inline tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature:205]:
```suggestion
#   exclude-groups-from-sharing configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (SwikritiT) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: Maybe issue tag of this https://github.com/owncloud/ocis/issues/1328 issue will also be relevant here?

Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: ISSUE TAG 1317 is present at the top and 1317 and 1328 are inter-related. I think this will overload issue tag

Comment (SwikritiT) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: Just because they are interrelated doesn't mean they are the same thing. If issue tag 1317 is at the top you can add 1328 alongside it. I'm just asking this for documentation as in issue https://github.com/owncloud/ocis/issues/1328#issuecomment-1184237905 it's mentioned here that it'll be implemented in ocis spaces too

**Review by SwikritiT — COMMENTED** [review]
LGTM

**Review by grgprarup — COMMENTED** [review]
LGTM :+1:

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (saw-jan) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]:
```suggestion
     # shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (saw-jan) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:41]:
```suggestion
     # shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (saw-jan) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:50]:
```suggestion
     # shares-only-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (saw-jan) [inline tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:58]:
```suggestion
     # allow-shares-with-groups-members configuration is not available yet in the oCIS webUI
```

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature
Comment (saw-jan) [inline tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature:205]:
```suggestion
     # exclude-groups-from-sharing configuration is not available yet in the oCIS webUI
```

**Review by saw-jan — COMMENTED** [review]
some indentation

File: tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature
Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: my IDE auto-generate this indentation. Does this auto-generate comment indentation wrong?

Comment (saw-jan) [reply tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature:32]: The comment is for the step below so I would expect that the comment aligns with the step's indentation.

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature
Comment (saw-jan) [inline tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature:3]: As we have come to conclusion, we restrain ourselves from adding comments in feature files
```suggestion
```

Comment (amrita-shrestha) [reply tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature]: I didn't make any changes in this PR after internal discussion

**Review by individual-it — APPROVED** [review]

**Pull request merged**
